public/static-checkout/index.html

```diff
@@ -0,0 +1,435 @@
+                      <option value="SE">Sweden</option>
+                      <option value="CH">Switzerland</option>
+                      <option value="SY">Syrian Arab Republic</option>
+                      <option value="TW">Taiwan</option>
+                      <option value="TJ">Tajikistan</option>
+                      <option value="TZ">Tanzania, United Republic of</option>
+                      <option value="TH">Thailand</option>
+                      <option value="TL">Timor-Leste</option>
+                      <option value="TG">Togo</option>
+                      <option value="TK">Tokelau</option>
+                      <option value="TO">Tonga</option>
+                      <option value="TT">Trinidad and Tobago</option>
+                      <option value="TN">Tunisia</option>
+                      <option value="TR">Turkey</option>
+                      <option value="TM">Turkmenistan</option>
+                      <option value="TC">Turks and Caicos Islands</option>
+                      <option value="TV">Tuvalu</option>
+                      <option value="UG">Uganda</option>
+                      <option value="UA">Ukraine</option>
+                      <option value="AE">United Arab Emirates</option>
+                      <option value="GB">United Kingdom</option>
+                      <option value="US">United States of America</option>
+                      <option value="UM">United States Minor Outlying Islands</option>
+                      <option value="UY">Uruguay</option>
+                      <option value="UZ">Uzbekistan</option>
+                      <option value="VU">Vanuatu</option>
+                      <option value="VE">Venezuela</option>
+                      <option value="VN">Viet Nam</option>
+                      <option value="VG">Virgin Islands, British</option>
+                      <option value="VI">Virgin Islands, U.S.</option>
+                      <option value="WF">Wallis and Futuna</option>
+                      <option value="EH">Western Sahara</option>
+                      <option value="YE">Yemen</option>
+                      <option value="ZM">Zambia</option>
+                      <option value="ZW">Zimbabwe</option>
+                      <option value="AX">Åland Islands</option>
+                      <option value="BQ">Bonaire, Sint Eustatius and Saba</option>
+                      <option value="CW">Curaçao</option>
+                      <option value="GG">Guernsey</option>
+                      <option value="IM">Isle of Man</option>
+                      <option value="JE">Jersey</option>
+                      <option value="ME">Montenegro</option>
+                      <option value="BL">Saint Barthélemy</option>
+                      <option value="MF">Saint Martin (French part)</option>
+                      <option value="RS">Serbia</option>
+                      <option value="SX">Sint Maarten (Dutch part)</option>
```

Comment:
spelling error 
it seems like we shouldn't need to enter any of these manually, i believe i included somewhere a list that i just got directly from the production drop down

Reply:
Spelling error - you mean Sint should be Saint? 
Emma decided it would be better to have them in the HTML already then how I did it before in the JS file as an object with a for loop.

Reply:
yup
agree they should be in the html, i just meant we don't have to do any of the html by hand, which could result the spelling errors

Reply:
sorry i incorrectly assumed Sint was supposed to be Saint (though i do think just programmatically copying the spellings from the dropdown on payment.edx.org might be ideal since it is different)